include/wasm/WasmCommon.h.in

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+/* The constants defined in this header file are meant to
+ * be shared across all supported WASM runtimes: WAVM, WAMR, and WAMR inside
+ * SGX. In particular, this header must also be included inside an SGX enclave
+ * so we want to keep the external includes to a minimum. Some of the values for
+ * this constants must match those used when cross-compiling WASM modules, as
+ * a consequence, we define them in faasmtools/build.py and populate the
+ * values using CMake
+ */
+
+#define WASM_BYTES_PER_PAGE @FAASM_WASM_BYTES_PER_PAGE@
```

Comment:
Is there a point to keeping bytes-per-page dynamic? Afaik this value is frozen at 64k in the wasm spec